docs/api-reference/interfaces/AssertViewOpts.mdx

```diff
@@ -0,0 +1,22 @@
+[testplane](../README.mdx) / AssertViewOpts
+
+# Interface: AssertViewOpts
+
+## Extends
+
+- `Partial`\<[`AssertViewOptsConfig`](AssertViewOptsConfig.mdx)\>
+
+## Properties
+
+| Property | Type | Default value | Description | Inherited from | Defined in |
+| ------ | ------ | ------ | ------ | ------ | ------ |
+| `allowViewportOverflow?` | `boolean` | `false` | Disables check that element is outside of the viewport left, top, right or bottom bounds. **Remarks** By default Testplane throws an error if element is outside the viewport bounds. This option disables check that element is outside of the viewport left, top, right or bottom bounds. And in this case if browser option [https://github.com/gemini-testing/testplane#compositeimage compositeImage](https://github.com/gemini-testing/testplane#compositeimage) set to `false`, then only visible part of the element will be captured. But if [https://github.com/gemini-testing/testplane#compositeimage compositeImage](https://github.com/gemini-testing/testplane#compositeimage) set to `true` (default), then in the resulting screenshot will appear the whole element with not visible parts outside of the bottom bounds of viewport. | `Partial.allowViewportOverflow` | [src/config/types.ts:45](https://github.com/gemini-testing/testplane/blob/57d0b182c37e21d8bdf52acbb137a028841767af/src/config/types.ts#L45) |
```

Comment:
Nice to have: is it possible to split it into multiple lines? Because now it's quite hard to read, when it all is on one line.